src/test/scala/fi/oph/scalaschema/ValidationAndExtractionTest.scala

```diff
@@ -24,15 +24,15 @@ class ValidationAndExtractionTest extends AnyFreeSpec with Matchers {
       }
       "Missing fields validation" in {
         verifyValidation[TestClass](JObject(), Left(List(
-          ValidationError("name",JNothing,MissingProperty()),
-          ValidationError("stuff",JNothing,MissingProperty())
+          ValidationError("name", JNothing, MissingProperty()),
```

Comment:
Commit history would be clearer especially for review, if this kind of whitespace changes and line splits were all done in a separate cleanup commit without any functionality changes.